src/snippets/interview.v1.yaml

```diff
@@ -20,6 +20,13 @@ properties:
     updated:
         title: Updated date (UTC)
         $ref: ../misc/date-time.v1.yaml
+    image:
+        type: object
+        properties:
+            thumbnail:
+                $ref: ../misc/image.v1.yaml
+        required:
+          - thumbnail
```

Comment:
It ia my understanding that if we have an optional image then we must have a thumbnail to serve an image to the listing page.

Similar to the requirement for article listings: https://github.com/elifesciences/api-raml/blob/develop/src/snippets/article-vor.v1.yaml#L22